dist/css/bootstrap.css

```diff
@@ -3987,7 +3987,7 @@ input[type="button"].btn-block {
 
 .card-header {
   padding: 0.75rem 1.25rem;
-  background-color: #f5f5f5;
+  background-color: whitesmoke;
```

Comment:
Well that's interesting. It doesn't do it for other colors that have names. Also, please remove the dist files as per CONTRIBUTING.md in order to reduce the risks of merge conflicts